### PR TITLE
Add Instagram auth

### PR DESCRIPTION
**Pull request by luanmuniz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,36 +84,44 @@ must be: `https://login.yourdomain.com/facebook/callback`
...
 ## Mixer specific environment variables
 
-You need to create your own Mixer OAuth Client. If `LW_SUBDOMAIN=login.yourdomain.com` your Authorization callback URL 
+You need to create your own Mixer OAuth Client. If `LW_SUBDOMAIN=login.yourdomain.com` your Authorization callback URL
 must be: `https://login.yourdomain.com/mixer/callback`
 
 - `LW_MIXER_CLIENTID` - Your Mixer Client ID
 - `LW_MIXER_CLIENTSECRET` - Your Mixer Client Secret
 
+## Mixer specific environment variables
```
Comment (lipp) [inline README.md]: this should be "instagram"

File: README.md [hunk 2]
```diff
@@ -83,36 +84,44 @@ must be: `https://login.yourdomain.com/facebook/callback`
...
+
+You need to create your own Instagram OAuth application. If `LW_SUBDOMAIN=login.yourdomain.com` your Authorization callback URL
+must be: `https://login.yourdomain.com/mixer/callback`
```
Comment (lipp) [inline README.md]: this should be `/instagram/callback`

File: src/strategies/instagram.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    done(null, {
+      token,
+      tokenSecret,
+      profile: {
+        username,
+        provider,
+        photo: _json.data.profile_picture,
+        name: displayName
+      },
+      meta: _json.data
```
Comment (lipp) [inline src/strategies/instagram.js]: the problem here is, that cookie size is pretty [limited](http://browsercookielimits.squawky.net/)

--> issue #36

**Comment by lipp** [comment]
THX 💯

**Comment by luanmuniz** [comment]
@lipp Done!

File: README.md [hunk 4]
```diff
@@ -123,6 +132,7 @@ must be: `https://login.yourdomain.com/mixer/callback`
 - `/reddit` - login with Reddit account (if configured through env variables)
 - `/mixer` - login with Mixer account (if configured through env variables)
 - `/linkedin` - login with LinkedIn account (if configured through env variables)
+- `/instagram` - login with LinkedIn account (if configured through env variables)
```
Comment (lipp) [inline README.md]: typo

**Comment by lipp** [comment]
great! will merge tomorrow.

File: README.md (hunk 4 above)
Comment (luanmuniz) [inline README.md]: My bad

**Pull request merged**